guides/setting-up.md

```diff
@@ -0,0 +1,40 @@
+---
+explains: A working guide for analists in the DataLab 
+---
+
+# Setting up
```

Comment:
I suggest using a more descriptive title such as 'Setting up an data analysis project', to give a bit more context

Reply:
Nice will change this.